components/pango_python/CMakeLists.txt

```diff
@@ -73,6 +78,12 @@ if(BUILD_PANGOLIN_PYTHON AND Python_FOUND AND pybind11_FOUND)
         $<INSTALL_INTERFACE:include>
     )
 
+    if (NOT APPLE)
+        set_target_properties(
+            pypangolin PROPERTIES VERSION ${PANGOLIN_VERSION} SOVERSION ${PANGOLIN_VERSION_MAJOR}
+        )
+    endif()
+
```

Comment:
Does this actually have an effect? Looking at the log, the generated so file:
> Linking CXX shared module pypangolin-0.9.1.data/purelib/pypangolin.cpython-310-x86_64-linux-gnu.so

does not contain the versioning. Additionally, this should not be required for the `pypangolin.cpython` since this is not dynamically linked. Are the `PROPERTIES VERSION` really required in the `pypangolin` component?

Reply:
@christian-rauch ah, you might be right. The pypangolin library does not need to be, but the embedded python does. I'll push an update after doing a test build.